src/notifications/notificationmanager.cpp

```diff
@@ -414,7 +414,7 @@ NotificationList NotificationManager::GetNotifications(const QString &owner)
     QHash<uint, LipstickNotification *>::const_iterator it = notifications.constBegin(), end = notifications.constEnd();
     for ( ; it != end; ++it) {
         LipstickNotification *notification = it.value();
-        if (notification->owner() == owner) {
+        if (notification && notification->owner() == owner) {
```

Comment:
...if there's a problem to begin with in this kind of places. Seems only newly created instances are inserted. Could be that only bug was trying to delete with non-existent id?